pkg/deployment/cert.go

```diff
@@ -109,7 +109,9 @@ func EnsureTLSCerts(ctx context.Context, helper *helper.Helper,
 
 		if service.Spec.TLSCert.Issuer == "" {
 			// by default, use the internal root CA
-			issuer = certmanager.RootCAIssuerInternalLabel
+			// issuer = certmanager.RootCAIssuerInternalLabel
+			// TODO(alee) Temporarily set this to the issuer name
+			issuer = "rootca-internal"
```

Comment:
I think we can instead use `tls.DefaultCAPrefix + endpoint.EndpointInternal` to align with the pattern used for the issuer name in openstack-operator, rather than hard-coding here...